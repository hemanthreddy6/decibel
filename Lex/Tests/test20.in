loop over i 0 to 5 @1 {
    audio loopedSegment <- repeatedEffect[i*0.5s : (i+1)*0.5s];
    audio scaledSegment <- loopedSegment * (i + 1);
    play scaledSegment;
}
