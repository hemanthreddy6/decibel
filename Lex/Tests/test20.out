LOOP
OVER
IDENTIFIER
INT_LITERAL
TO
INT_LITERAL
@
INT_LITERAL
{
AUDIO
IDENTIFIER
<-
IDENTIFIER
[
IDENTIFIER
*
FLOAT_LITERAL
:
(
IDENTIFIER
+
INT_LITERAL
)
*
FLOAT_LITERAL
]
;
AUDIO
IDENTIFIER
<-
IDENTIFIER
*
(
IDENTIFIER
+
INT_LITERAL
)
;
PLAY
IDENTIFIER
;
}
